features/number.yml

```
@@ -0,0 +1,30 @@
+name: Number
+description: The Number object is used to store number and manipulate floating-point numbers, like 42, or -4.201
+spec: https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-number-objects
+status:
+  compute_from: javascript.builtins.Number
+compat_features:
+  - javascript.builtins.Number
```

Since `typeof NaN` and `typeof Infinity` → `"number"`, I think it follows that we should include them here too:

```suggestion
  - javascript.builtins.Infinity
  - javascript.builtins.NaN
  - javascript.builtins.isNaN
  - javascript.builtins.Number
```